app/src/main/java/com/cornellappdev/android/eatery/AccountInfoFragment.java

```diff
@@ -0,0 +1,86 @@
+package com.cornellappdev.android.eatery;
+
+import android.os.Bundle;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentTransaction;
+import android.view.LayoutInflater;
+import android.view.Menu;
+import android.view.MenuInflater;
+import android.view.MenuItem;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ListView;
+import android.widget.TextView;
+
+import com.cornellappdev.android.eatery.model.BrbInfoModel;
+import com.cornellappdev.android.eatery.util.MoneyUtil;
+
+public class AccountInfoFragment extends Fragment {
+
+    private TextView mSwipesLabel;
+    private TextView mBrbLabel;
+    private TextView mCityBucksLabel;
+    private TextView mLaundryLabel;
+    private HistoryInfoAdapter mListAdapter;
+    private ListView mHistoryView;
+    private View mInfoHeader;
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+            Bundle savedInstanceState) {
+        View rootView = inflater.inflate(R.layout.fragment_account_info, container, false);
+        mHistoryView = rootView.findViewById(R.id.historylistview);
+        mInfoHeader = inflater.inflate(R.layout.account_info_header, null);
+        mSwipesLabel = mInfoHeader.findViewById(R.id.swipesValue);
+        mBrbLabel = mInfoHeader.findViewById(R.id.brbValue);
+        mCityBucksLabel = mInfoHeader.findViewById(R.id.cityBucksValue);
+        mLaundryLabel = mInfoHeader.findViewById(R.id.laundryValue);
+
+        BrbInfoModel model = Repository.getInstance().getBrbInfoModel();
+        String outputText = "";
+        if (model.getSwipes() == 1) {
+            outputText = model.getSwipes() + " meal left";
+        } else {
+            outputText = model.getSwipes() + " meals left";
+        }
+        mSwipesLabel.setText(outputText);
+        mBrbLabel.setText(MoneyUtil.toMoneyString(model.getBRBs()));
+        mCityBucksLabel.setText(MoneyUtil.toMoneyString(model.getCityBucks()));
+        mLaundryLabel.setText(MoneyUtil.toMoneyString(model.getLaundry()));
+
+        mListAdapter = new HistoryInfoAdapter(getContext(), R.layout.history_item,
```

Comment:
from my understanding, this list adapter contains the list of past items purchased. how many list items do you display? if you display over 15 items, you should use a `RecyclerView`

Reply:
@connorreinhold please don't attempt to change the list into a recyclerview in this pr!! make an issue instead

Reply:
Raised the issue